src/components/CurrentUser.js

```diff
@@ -30,14 +30,14 @@ const UserStatus = styled.div`
   font-size: 0.9rem;
 `;
 
-export default function ActiveUser({ activeUser, handleActiveUserClick }) {
+export default function CurrentUser({ currentUser, handleCurrentUserClick }) {
```

Comment:
prop이름은 이벤트 핸들러로 하지 않아요.

```suggestion
export default function CurrentUser({ currentUser, onClick }) {
```

이벤트 핸들러가 들어가야 하는 prop자리의 명칭은 on[EventName]으로 하는 것이 일반적입니다.

그래야 상위 컴포넌트에서
```jsx
<CurrentUser onClick={handleCurrentUserClick} />
```

이렇게 쓸 수 있는거니까요~!